src/main/java/io/anserini/index/generator/WapoGenerator.java

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
I know it's long a verbose, but consider renaming `WashingtonPostDocumentGenerator`?